### PR TITLE
Modify post to select random default image

### PR DESCRIPTION
**Pull request by kharazzoro**

### CODE REVIEW

File: web/src/components/Post.js [hunk 1]
```diff
@@ -31,14 +28,34 @@ export default class Post extends Component {
...
     postID: null,
     updateButton: false,
     imageLoadError: null,
     postTypeDispalyText: '',
     postError: null
   };
+
+  componentDidMount(){
+ 
+    let Images={
```
Comment (kkarimi) [inline web/src/components/Post.js]: 1) Best to take this out this code to a new function called `getRandomImage()`
2) Please install and use `prettier` in your editor, spacing currently is not consistent

Comment (kharazzoro) [reply web/src/components/Post.js]: updated

Comment (kkarimi) [reply web/src/components/Post.js]: This whole section of code does not belong in `componentDidMount`, it needs to be taken out and you call that function from here.

File: web/src/components/Post.js [hunk 2]
```diff
@@ -31,14 +28,44 @@ export default class Post extends Component {
...
+    5:'https://www.planwallpaper.com/static/images/background-gmail-google-images_FG2XwaO.jpg',
+    6:'https://www.w3schools.com/w3css/img_fjords.jpg',
+    7:'https://www.w3schools.com/w3css/img_lights.jpg',
+    8:'https://www.elastic.co/assets/bltada7771f270d08f6/enhanced-buzz-1492-1379411828-15.jpg',
+    9:'http://images.all-free-download.com/images/graphiclarge/canoe_water_nature_221611.jpg',
+    10:'http://images.all-free-download.com/images/graphiclarge/landscape_meadow_nature_216362.jpg' 
+    }
+       
+    const getRandomImage=()=>{
+      let imagesLength=Object.keys(Images).length ;
```
Comment (kkarimi) [inline web/src/components/Post.js]: You still have not configured prettier from previous comment

**Comment by kharazzoro** [comment]
@kkarimi Updated

**Pull request merged**